src/services/network-service.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const API_URL = "http://server.jaram.net:5905/";
+
+class NetWorkService {
```

Comment:
If this module will use later code snipets, we need to additional parsing function from server's basic response form.
Our API Document will be helpful.